packages/features/form-builder/FormBuilder.tsx

```diff
@@ -44,11 +44,13 @@ export const FormBuilder = function FormBuilder({
   description,
   addFieldLabel,
   formProp,
+  shouldDisplayLocation,
```

Comment:
FormBuilder can't have hardcoded info for a field. It is a general purpose component and it is supposed to power Routing Forms as well.

Reply:
We can possibly solve it by supporting a field like `hideWhenJustOneOption` and setting it to true for location field.

Reply:
okay i added this field hideWhenJustOneOption but now if i add more location and click save and go to advanced tab. i would have to reload to make sure location field is displayed.